.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      TAG_NAME: ${{ github.event.inputs.tag }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: 17
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+        with:
+          gradle-version: 8.5
+      - name: Build with Gradle
+        id: build
+        run: |
+          gradle packageRelease
+          echo zip_path=`find ./release/*.zip` >> $GITHUB_OUTPUT
+      - name: Release
+        uses: softprops/action-gh-release@v2
```

Comment:
Great work! 🎉
 just a quick note: `action-gh-release` supports file globbing so `files: release/*.zip` would probably work and results in easier to read code

Reply:
Thank you for the suggestion, @Stefterv ! You're right, using globbing would make it easier to understand. I’ll incorporate that 👍